### PR TITLE
fix: trashy spawn points feat: locales

### PR DESCRIPTION
**Pull request by artur-michalak**
## Description

Fix:
https://github.com/Qbox-project/qbx_spawn/issues/17

Locales:
obraz

## Checklist

- [X] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [X] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -32,8 +34,8 @@ local function managePlayer()
 end
 
 local function createSpawnArea()
-    for i = 1, #config.spawns, 1 do
-        local spawn = config.spawns[i]
+    for i = 1, #spawns, 1 do
```
Comment (Manason) [inline client/main.lua:37]:
```suggestion
    for i = 1, #spawns do
```

File: client/main.lua [hunk 2]
```diff
@@ -156,7 +159,7 @@ end
 local function updateScaleform()
     if previousButtonID == currentButtonID then return end
 
-    for i = 1, #config.spawns, 1 do
+    for i = 1, #spawns, 1 do
```
Comment (Manason) [inline client/main.lua:162]:
```suggestion
    for i = 1, #spawns do
```

File: config/client.lua [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
             coords = vec4(195.17, -933.77, 29.7, 144.5)
         },
         {
-            label = 'Paleto Bay',
+            label = 'paleto_bay',
             coords = vec4(80.35, 6424.12, 31.67, 45.5),
         },
         {
-            label = 'Motels',
+            label = 'motels',
```
Comment (Manason) [inline config/client.lua]: Why these changes?

**Review by Manason — CHANGES_REQUESTED** [review]

File: config/client.lua (hunk 3 above)
Comment (artur-michalak) [reply config/client.lua]: not in every language these places have the same names

Comment (BerkieBb) [reply config/client.lua]: Configs shouldn't hold locales, they should hold values that you can change only. These ones are not meant to be changed so they are not meant for the config.

Comment (Manason) [reply config/client.lua]: In this case, I don't think locales are appropriate as this is a highly configurable and dynamic array. If someone wants to add a new location they will then need to add it to the locales files as well.

Comment (artur-michalak) [reply config/client.lua]: What do you think about such a concept?
en.json
```json
{
    "last_location": "Last Location",
}
```
pl.json
```json
{
    "last_location": "Ostatnia pozycja",
    "Legion Square": "Kwadraciak",
    "_Paleto Bay": "removed item placeholder",
    "Motels": "Motele"
}
```

Comment (Manason) [reply config/client.lua]: Are you trying to support client side locales? If not, I think the least amount of work for server owners will just be to edit these in config only and not bother with the locales file.

**Review by Manason — APPROVED** [review]

File: config/client.lua (hunk 3 above)
Comment (artur-michalak) [reply config/client.lua]: Exactly
